common/pulp_docker/common/tarutils.py

```diff
@@ -114,3 +114,24 @@ def get_youngest_children(metadata):
                 pass
 
     return list(image_ids)
+
+
+def get_image_manifest(tarfile_path):
+    """
+    Given a path to a tarfile, which itself is the product of "skopeo copy"
+    this returns the decoded manifest.json file if it exists.
+
+    :param tarfile_path: full path to tarfile
+    :type tarfile_path:  basestring
+    :return:             decoded manifest json
+    :rtype               list or dict
+    """
+    image_manifest = []
```

Comment:
When you save a docker image using docker save command, it does create a manifest file as well. And since the manifest file in this case is an array of objects, I am checking whether the decoded manifest json is a list or a dict to verify whether it is a V1 or V2.